.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.6"
```

Comment:
should that be 3.6.1?? or does that not matter? Im a python noob

Reply:
Ideally it should be 3.6.3 as that's what we specify in the pipfile but shouldn't be a major issue short term. Need to see if we can do that with travos

Reply:
You're right it should have been 3.6.1